### ISSUE TITLE
[FORMATTING] MariaDB formats sample query on demo page differently

### ISSUE THREAD

**Issue by verhovsky** [sql-formatter-org/sql-formatter#298]
**Input data**
```sql
select supplier_name,city from
(select * from suppliers join addresses on suppliers.address_id=addresses.id)
as suppliers
where supplier_id>500
order by supplier_name asc,city desc;
```

with the default settings on the website.

**Expected Output**
Every other formatter formats it like this:

```sql
select
  supplier_name,
  city
from
  (
    select
      *
    from
      suppliers
      join addresses on suppliers.address_id = addresses.id
  ) as suppliers
where
  supplier_id > 500
order by
  supplier_name asc,
  city desc;
```

**Actual Output**

The MariaDB formatter formats it like this:

```sql
select
  supplier_name,
  city
from
  (
    select
      *
    from
      suppliers
      join addresses on suppliers.address_id = addresses.id
  ) as suppliers
where
  supplier_id > 500
order by
  supplier_name asc,
  city
desc
;
```

**Usage**

- How are you calling / using the library?
https://sql-formatter-org.github.io/sql-formatter/
- What SQL language(s) does this apply to?
MariaDB
- Which SQL Formatter version are you using?
`latest`

This happens because in MariaDB, `DESC` is short for `DESCRIBE` so it appears in `reservedCommands` instead of `reservedKeywords` 

https://github.com/sql-formatter-org/sql-formatter/blob/76fb498acbc0e479eaeb3fb299045eaa8b619d3d/src/languages/mariadb.formatter.ts#L939

**Comment by nene** [comment]
Thanks for a detailed bug report. Fixed now.

**Comment by nene** [comment]
Fix released in 8.1.0
